src/main/java/edu/cornell/kfs/module/purap/document/service/impl/PurchaseOrderTransmissionMethodDataRulesServiceImpl.java

```diff
@@ -16,14 +15,16 @@
  */
 public class PurchaseOrderTransmissionMethodDataRulesServiceImpl implements PurchaseOrderTransmissionMethodDataRulesService{
 	
+	private PostalCodeValidationService postalCodeValidationService;

```

Comment:
The getters and setters are public, shouldn't that provide the needed capability to override?